rust/operator-binary/src/config.rs

```diff
@@ -16,44 +16,46 @@ pub const PYTHON_IMPORTS: &[&str] = &[
 
 pub fn add_airflow_config(
     config: &mut BTreeMap<String, String>,
-    authentication_config: Option<&AirflowClusterAuthenticationConfig>,
-    authentication_class: Option<&AuthenticationClass>,
+    authentication_config: &Vec<AirflowAuthenticationConfigResolved>,
 ) {
-    if let Some(authentication_config) = authentication_config {
-        if let Some(authentication_class) = authentication_class {
-            append_authentication_config(config, authentication_config, authentication_class);
-        }
-    }
     if !config.contains_key(&*AirflowConfigOptions::AuthType.to_string()) {
         config.insert(
             // should default to AUTH_TYPE = AUTH_DB
             AirflowConfigOptions::AuthType.to_string(),
             "AUTH_DB".into(),
         );
     }
+
+    append_authentication_config(config, authentication_config);
 }
 
 fn append_authentication_config(
     config: &mut BTreeMap<String, String>,
-    authentication_config: &AirflowClusterAuthenticationConfig,
-    authentication_class: &AuthenticationClass,
+    authentication_config: &Vec<AirflowAuthenticationConfigResolved>,
 ) {
-    if let AuthenticationClassProvider::Ldap(ldap) = &authentication_class.spec.provider {
-        append_ldap_config(config, ldap);
-    }
+    // TODO: we make sure in crd/src/authentication.rs that currently there is only one
```

Comment:
Would this comment be more visible if we moved it to AirflowClusterConfig?

Reply:
I like to keep the todos, but ill add the size limit comment to AirflowClusterConfig for the CRD description?

Reply:
Agreed.